index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Dancing+Script&family=Roboto:wght@700&family=Zen+Dots&display=swap" rel="stylesheet">
+</head>
+<body>
+    <main class="wrapper">
+    <header>
+        <h1>My Logo</h1>
+        <img src="Union.png" alt="navigation icon">
+    </header>
+
+    <section class="section-one">
+        <h1>I'm Salma Glad to see you!</h1>
+        <p>I'm a software developer! 
+            I can help you build a product , 
+            feature or website Look through 
+            some of my work and experience! 
+            If you like what you see and have
+             a project you need coded, 
+            don't hestiate to contact me.</p>
+            <h3>LET'S CONNECT</h3>
+            <img src="ic-social media icons.png" alt="social-media icons">
+    </section> 
+</main>
+</body>
```

Comment:
- I'd like to kindly suggest that we stick to the Figma template provided as much as possible for this portfolio project. This includes ;
`background images`, 
`font sizes`, 
`font colors`, 
`font family`,
`hamburger menu`,
`social icons`
and so on. 

- Can we refactor your code to achieve the above mentioned?? 

Here is what you have compared to what's on Figma 👇🏽 

| Your Design  |Figma Design |
| :---:   | :---: | 
| <img width="383" alt="Screenshot 2023-01-12 at 00 38 10" src="https://user-images.githubusercontent.com/99246359/211995426-2c49e17c-510b-4e1f-ac5d-593689d52c89.png">|<img width="374" alt="Screenshot 2023-01-12 at 00 19 39" src="https://user-images.githubusercontent.com/99246359/211995558-22d3cba7-f694-4ec9-ae79-deebe2075c37.png">  |